### DISCUSSION TITLE
Migration "issues"

### DISCUSSION THREAD

**Discussion by tzwaeaen** [Mailu/Mailu#3324] [User support]
Hello everyone,

I migrated to 2024.06 but I'm having some small issues. I followed the steps in the release note and created a config with the setup tool.

When using TLS_FLAVOR=letsencrypt I get an error when spinning up the stack.

> Missing cert or key file, disabling TLS
doveconf: Fatal: Error in configuration file /etc/dovecot/proxy.conf line 38: ssl_cert: Can't open file /certs/letsencrypt/live/mailu/nginx-chain.pem: No such file or directory
...
CRITICAL:root:Can't reach http://mail.example.com/.well-known/acme-challenge/testing!, please ensure it's fixed or change the TLS_FLAVOR.

When I use TLS_FLAVOR=mail as before everything runs smooth.

I expected mailu to request/create the certs when there are none.

**Comment by nextgens** [comment]
Please double check that you have opened or proxied port 80/http. That is what is used to obtain the certificates from the CA.

**Reply by tzwaeaen** [reply]
It was this part that I skipped on the front container.  Added it, front starts and requests the certs. Can access the admin panel but unable to connect to the mailserver. :-|

>   healthcheck:
    test: ['NONE']

**Reply by nextgens** [reply]
Describe what you are attempting, what is happening and what you were expecting if you need help.

Logs may also be helpful.
